avocado/utils/process.py

```diff
@@ -377,12 +377,8 @@ def _init_subprocess(self):
                                                shell=self.shell,
                                                env=self.env)
             except OSError as details:
-                if details.errno == 2:
-                    exc = OSError("File '%s' not found" % self.cmd.split()[0])
-                    exc.errno = 2
-                    raise exc
-                else:
-                    raise
+                details.strerror += " (%s)" % self.cmd
+                raise details
```

Comment:
Yep, this includes the cmd in any case along with the original exception, therefor we can not lose any information, we can only see a duplicate information in case the original exception includes the cmd. But standard python libraries don't do that (for whatever reason)